executor/builder.go

```diff
@@ -2582,13 +2590,20 @@ func (b *executorBuilder) buildAnalyzeColumnsPushdown(task plannercore.AnalyzeCo
 	failpoint.Inject("injectAnalyzeSnapshot", func(val failpoint.Value) {
 		startTS = uint64(val.(int))
 	})
+
+	isAutoFlag := uint64(0)
+
+	if autoAnalyze != "" {
```

Comment:
Any test to cover this case?

Reply:
No test case for it yet. I tested it manually at TiKV server side.